.azure-pipelines/stages.yml

```diff
@@ -57,6 +57,7 @@ stages:
   jobs:
   - template: env.yml
 
+
```

Comment:
Should this be removed?

Reply:
probably yes - i will do it in one of my follow up prs rather than running ci again for ws